dashboard/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "dashboard",
   "version": "0.1.0",
   "private": true,
+  "proxy": "https://localhost", 
```

Comment:
Why this change?

Reply:
Oops, committed by mistake. It is what I use for local UI development :facepalm: 
Deleting the change straight away :S